projects/valtimo/config/assets/core/nl.json

```diff
@@ -165,7 +165,21 @@
   "formManagement": {
     "noFormDefinitionFoundAdmin": "Er is geen formulier gekoppeld aan de start event/user task. Om een formulier te koppelen ga naar Admin menu > Proceskoppelingen",
     "noFormDefinitionFoundUser": "Er is geen formulier gekoppeld aan de start event/user task. Vraag een admin gebruiker om hulp.",
-    "gotoFormLinksButton": "Configureer nu"
+    "gotoFormLinksButton": "Configureer nu",
+    "name": "Naam",
+    "nameIsRequired": "Naam is vereist",
+    "back": "Ga terug",
+    "reset": "Resetten",
+    "submit": "Indienen",
+    "delete": "Verwijderen",
+    "deploy": "Deploy",
+    "readOnly": "Alleen-lezen"
```

Comment:
Is de - nodig?

Reply:
https://support.microsoft.com/nl-nl/office/een-bericht-tonen-om-een-bestand-als-alleen-lezen-te-openen-5c25909c-46d9-4eb0-9d1f-d072a560e340 yep